util/check_tool_requirements.py

```diff
@@ -96,8 +124,13 @@ def pip3_get_version(tool):
     return None
 
 
-def check_version(requirements, tool_name, getter):
+def check_version(requirements, tool_name, getter, version_converter = None):
     required_version = requirements.get(tool_name)
+    # If a version string converter is defined, call it. This is required
+    # for some version strings that are not compatible with StrictVersion.
+    if version_converter is not None:
+        required_version = version_converter(required_version)
```

Comment:
Do we expect `version_converter` to deal with a `None` argument? If not (and we're keeping this code), this should probably move after the next check.

Reply:
Oh yeah, this is wrong, thanks for catching that.